package.json

```diff
@@ -8,12 +8,13 @@
   "dependencies": {
     "@fastify/accept-negotiator": "^1.1.0",
     "fastify-plugin": "^4.5.0",
-    "into-stream": "^6.0.0",
     "mime-db": "^1.52.0",
     "minipass": "^7.0.2",
     "peek-stream": "^1.1.3",
     "pump": "^3.0.0",
-    "pumpify": "^2.0.1"
+    "pumpify": "^2.0.1",
+    "readable-stream": "^4.5.2",
```

Comment:
Why do you want to add the dep if it works on all release lines?

Reply:
`readable-stream` provides a stable copy of `stream` that prevent strange behavior across different node version.